### ISSUE TITLE
Data helper fails to create several databases

### ISSUE THREAD

**Issue by DespaxJulien** [architecture-building-systems/CityEnergyAnalyst#2123]
Hi, 

I've been trying to create a new project from the 2.16 Dashboard. When I run the `data helper`, it returns an error and some databases are not created.

Here are the steps to reproduce the error:

1/ Create new scenario from the dashboard: choose to create zone, district and terrain. Select random and simple buildings,
2/ Run `data helper` with all databases selected. The dashboard console returns this error:

```
City Energy Analyst version 2.16
Running `cea data-helper` with the following parameters:
- general:scenario = I:\Logiciels\City Energy Analyst\test_terrain\terrain
  (default: C:\reference-case-open\baseline)
- data-helper:region = CH
  (default: CH)
- data-helper:databases = ['comfort', 'architecture', 'HVAC', 'internal-loads', 'supply', 'restrictions', 'technology']
  (default: ['comfort', 'architecture', 'HVAC', 'internal-loads', 'supply', 'restrictions', 'technology'])
Running data-helper with scenario = I:\Logiciels\City Energy Analyst\test_terrain\terrain
Running data-helper with archetypes = ['comfort', 'architecture', 'HVAC', 'internal-loads', 'supply', 'restrictions', 'technology']
Process Process-4:
Traceback (most recent call last):
  File "C:\Users\jdespax\Documents\CityEnergyAnalyst\Dependencies\Python\lib\multiprocessing\process.py", line 267, in _bootstrap
    self.run()
  File "C:\Users\jdespax\Documents\CityEnergyAnalyst\Dependencies\Python\lib\multiprocessing\process.py", line 114, in run
    self._target(*self._args, **self._kwargs)
  File "c:\users\jdespax\documents\cityenergyanalyst\cityenergyanalyst\cea\interfaces\dashboard\tools\worker.py", line 19, in run_script
    script(**kwargs)
  File "c:\users\jdespax\documents\cityenergyanalyst\cityenergyanalyst\cea\api.py", line 28, in script_runner
    script_module.main(config)
  File "c:\users\jdespax\documents\cityenergyanalyst\cityenergyanalyst\cea\datamanagement\data_helper.py", line 421, in main
    technology_flag=technology_flag)
  File "c:\users\jdespax\documents\cityenergyanalyst\cityenergyanalyst\cea\datamanagement\data_helper.py", line 92, in data_helper
    archetypes_schedules = pd.read_excel(locator.get_archetypes_schedules(), use, index_col=0).T
  File "C:\Users\jdespax\Documents\CityEnergyAnalyst\Dependencies\Python\lib\site-packages\pandas\util\_decorators.py", line 178, in wrapper
    return func(*args, **kwargs)
  File "C:\Users\jdespax\Documents\CityEnergyAnalyst\Dependencies\Python\lib\site-packages\pandas\util\_decorators.py", line 178, in wrapper
    return func(*args, **kwargs)
  File "C:\Users\jdespax\Documents\CityEnergyAnalyst\Dependencies\Python\lib\site-packages\pandas\io\excel.py", line 329, in read_excel
    **kwds)
  File "C:\Users\jdespax\Documents\CityEnergyAnalyst\Dependencies\Python\lib\site-packages\pandas\io\excel.py", line 456, in parse
    **kwds)
  File "C:\Users\jdespax\Documents\CityEnergyAnalyst\Dependencies\Python\lib\site-packages\pandas\io\excel.py", line 607, in _parse_excel
    sheet = self.book.sheet_by_name(asheetname)
  File "C:\Users\jdespax\Documents\CityEnergyAnalyst\Dependencies\Python\lib\site-packages\xlrd\book.py", line 476, in sheet_by_name
    raise XLRDError('No sheet named <%r>' % sheet_name)
XLRDError: No sheet named 
```
When I go the the scenario folder, I see that:
- The `technology `databases have been created
- All the databases of the `building-properties` folder are missing (with the exception of age and occupancy, who where already there before running `data helper`). 

Do you know how to solve this?
Thank you!

**Comment by reyery** [comment]
The error is due to the introduction of the 'UNIVERSITY' occupancy type in #2034, where it found in the `SG` database and not in the `CH` database.

Since `zone-helper` creates the `occupancy` file based on `COLUMNS_ZONE_OCCUPANCY`, it will contain the `UNIVERSITY` column.
https://github.com/architecture-building-systems/CityEnergyAnalyst/blob/dc367abcb52b20cde1d141ef808851da924bac9f/cea/datamanagement/zone_helper.py#L144-L148

So if the user decides to create `zone` using `zone-helper` and then use `data-helper` with the default database parameter `CH`, it would not be able to find `UNIVERSITY` in the database.

We could either:
-  Check for the names in the sheet names of selected database instead of `COLUMNS_ZONE_OCCUPANCY`
here. But it would result in ignoring any columns that are not found in the database
https://github.com/architecture-building-systems/CityEnergyAnalyst/blob/dc367abcb52b20cde1d141ef808851da924bac9f/cea/datamanagement/data_helper.py#L73-L81
OR

- Check for `COLUMNS_ZONE_OCCUPANCY` first and then check for the sheet names so that we can provide a different message, for the case of a valid occupancy type but it does not exist in the selected database. But this will throw an error if the `zone` file is created by the `zone-helper` and choosing `CH` database.
```
for name in columns: 
     if name in COLUMNS_ZONE_OCCUPANCY: 
         if name in database_sheets:
             ...
         else:
             # message for not in database
     else:
         # message for invalid column
```
OR

- Changing how `zone-helper` creates the columns by using the sheets of a selected database instead of 
`COLUMNS_ZONE_OCCUPANCY`.

I'm not too sure how to handle this, any thoughts? @gabriel-happle @daren-thomas @JIMENOFONSECA

**Comment by daren-thomas** [comment]
@DespaxJulien this was fixed and merged to master as part of PR #2145.

To get it to work with your scenario, either start again, or, delete the "technologies/archetypes" folder in your scenario inputs. I'm closing this for now, but feel free to re-open if it doesn't fix your problem.
